The adaptive sider and navbar were made

AlekseiKolesnikov opened the pull request:
I have made an adaptive sider with a navbar inside. I would like to have comments about the way that I have chosen to realize it, especially the array of icon elements.

AlekseiKolesnikov committed:
The adaptive sider and navbar were made

ShadowZilot commented on src/components/layout/AppHeader/AppHeader.tsx:
```diff
@@ -1,10 +1,67 @@
+                        "header__logo-container space-between-row-center-flex menu-is-hovered" :
+                        "header__logo-container space-between-row-center-flex"
+                    }
+                    onMouseOver={() => {
+                        if (!useWindowSize.getState().isDesktopScreen) {
+                            setIsDropDownMenuVisible(true)
+                        }
+                    }}
+                    onMouseOut={() => {
+                        setIsDropDownMenuVisible(false)
+                    }}
```
How do you handle case when user try open drop down menu from mobile device?

AlekseiKolesnikov replied:
Will be solved in the next contribution

ShadowZilot commented on src/components/layout/AppLayout/AppLayout.css:
```diff
@@ -7,7 +7,7 @@
 .layout-content-sider-container {
     width: var(--desktop-content-container-width);
-    height: calc(100vh - 230px);
+    height: calc(100vh - 222px);
```
magic constant?

AlekseiKolesnikov committed:
1. Previous comment about variables was resolved
2. Sliding in and out slider for mobile version of app was made, check the way of that solution
3. The drop-down menu was fixed and no it's just the toggle of the class, not a separate element
